build/go.bzl

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A wrapper for go_library() which adds .gofiles, .sourcefiles filegroup targets.
+#
+# Usage:
+#   replace lines like:
```

Comment:
won't this be endlessly fighting gazelle?

Reply:
Jay tells me it shouldn't be a problem so long as there is a go_library it doesn't care where it is imported from.

Will probably need some fixing before being used though